### DISCUSSION TITLE
fork 本项目 部署成功后没有 CODE输入框

### DISCUSSION THREAD

**Discussion by aiallincc** [ChatGPTNextWeb/NextChat#701] [Q&A]
![image](https://user-images.githubusercontent.com/130343760/231035918-e3827353-51d0-4c24-9c2c-4e63be74474f.png)

环境变量已经手动加入OPENAI_API_KEY 和CODE

**Comment by addlistener** [comment]
我也出现了，localStorage 里面你看一下 needCode 是不是 true

**Comment by Yidadaa** [comment]
需要重新设置环境变量，具体请查阅文档。
